core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -301,6 +301,28 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
     if (proxyUser != null && principal != null) {
       error("Only one of --proxy-user or --principal can be provided.")
     }
+
+    // Ensure heartbeat arguments is valid.
+    val executorTimeoutMs =
+      Utils.timeStringAsMs(sparkProperties.getOrElse(
+        "spark.storage.blockManagerSlaveTimeoutMs",
+        s"${Utils.timeStringAsSeconds(
+          sparkProperties.getOrElse("spark.network.timeout", "120s"))}s"))
+    val executorHeartbeatIntervalMs = Utils.timeStringAsMs(
+      sparkProperties.getOrElse("spark.executor.heartbeatInterval", "10s"))
+    val checkTimeoutIntervalMs =
```

Comment:
Could we please rename this to `checkNetworkTimeoutIntervalMs` ?

Reply:
It's ok! thanks